src/module/reservation.js

```diff
@@ -0,0 +1,139 @@
+const showReservations = (e) => {
+  const popup = document.createElement('div');
+  document.body.appendChild(popup);
+  popup.classList.add('popUp', 'popUpContainer');
+
+  const idMeal = e.target.parentElement.parentElement.children[0].children[0].innerHTML;
+  const url = `https://www.themealdb.com/api/json/v1/1/lookup.php?i=${idMeal}`;
+  const getData = async () => {
+    const response = await fetch(url);
+    const data = await response.json();
+    return data;
+  };
+  const lookup = getData();
+  lookup.then((data) => {
+    const mealName = data.meals[0].strMeal;
+    const mealImage = data.meals[0].strMealThumb;
+    const tags = data.meals[0].strTags.split(',');
+    popup.classList.add('popup');
+    popup.innerHTML = `
+    <img class = "popup-Images" src="${mealImage}" alt="Food Image">
+    <div class="popup-inner" id="start">
+        <article class="popup-header">
+            <button id="end"><svg class ="endALl" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512"><!--! Font Awesome Pro 6.2.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2022 Fonticons, Inc. --><path d="M310.6 150.6c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0L160 210.7 54.6 105.4c-12.5-12.5-32.8-12.5-45.3 0s-12.5 32.8 0 45.3L114.7 256 9.4 361.4c-12.5 12.5-12.5 32.8 0 45.3s32.8 12.5 45.3 0L160 301.3 265.4 406.6c12.5 12.5 32.8 12.5 45.3 0s12.5-32.8 0-45.3L205.3 256 310.6 150.6z"/></svg></button>
+        </article>
+        <article class="popup-body">
+            <h2 class ="mealName">${mealName}</h2>
+            <div class="card-tags">
+                <ul class = "popDet">
+                <li>Country : ${data.meals[0].strArea}</li>
+                <li>Catogory : ${data.meals[0].strCategory}</li>
+                </ul>
+                <ul class = "popDet2">
+                 <div class = "popDet2Head">Tags</div>
+                </ul>
+            </div>
+       </article>
+       <article class = "popup-footer">
+            <form class="form-inline" id = "makeRes">
+                 <h3 id ="headForm"> Make a Reservation </h3>
+                <input type="name" class="form-control" placeholder="Your Name" !required></input>
+                <label for="date">Start Date</label>
+                <input type="date" class="form-control" id="start-date" !required></input>
+                <label for="date">End Date</label>
+                <input type = "date" class="form-control" id="end-date" !required></input>
+                <button type = "submit" class="btn btn-primary" id="submitReservation">Reserve!</button>
+            </form>
+        </article>
+    `;
+    const popDet2 = document.querySelector('.popDet2');
```

Comment:
[OPTIONAL] Great work with the reservations feature, congrats on it! Just for the sake of UX, I would suggest trying an inline form, this way you will save up some vertical space to display more reservations as this is a really cool project for your portfolio:

|Inline Fom|Your design|
|---|---|
|![Screen Shot 2022-09-02 at 10 16 06 AM](https://user-images.githubusercontent.com/83717285/188183797-2723d341-ccfb-46b3-8ea9-ed32351e5bb3.png)|<img width="701" alt="Screen Shot 2022-09-02 at 10 15 26 AM" src="https://user-images.githubusercontent.com/83717285/188183861-75309aa6-0420-43fb-89b1-dd6c162bd73e.png">|